mkdocs/__main__.py

```diff
@@ -107,13 +108,16 @@ def callback(ctx, param, value):
     click.option('--use-directory-urls/--no-directory-urls', is_flag=True, default=None, help=use_directory_urls_help)
 ])
 
-pgk_dir = os.path.dirname(os.path.abspath(__file__))
+PKG_DIR = os.path.dirname(os.path.abspath(__file__))
 
 
 @click.group(context_settings={'help_option_names': ['-h', '--help']})
 @click.version_option(
-    '{} from {} (Python {})'.format(__version__, pgk_dir, sys.version[:3]),
-    '-V', '--version')
+    __version__,
+    '-V', '--version',
+    message='%(prog)s, version %(version)s from {} (Python {})'.format(
+        PKG_DIR, PYTHON_VERSION)
```

Comment:
I'm assuming the message argument is a more recent addition to click. In any event, I guess there is no reason to not make the change.

However, as we are making a change here, we might as well go all the way. As all supported Python versions now include support for f-strings, lets use them. They're not only easier to read, but they are more performant.

```python
message=f'%(prog)s, version %(version)s from { PKG_DIR } (Python { PYTHON_VERSION })'
```